packages/stripe_platform_interface/lib/src/models/payment_methods.dart

```diff
@@ -383,6 +383,42 @@ class PaymentMethodParams with _$PaymentMethodParams {
     required bool testOfflineBank,
   }) = _PaymentMethodParamsFpx;
 
+  @JsonSerializable(explicitToJson: true)
+  @FreezedUnionValue('SepaDebit')
+  const factory PaymentMethodParams.sepaDebit({
+    required String iban,
+    PaymentIntentsFutureUsage? setupFutureUsage,
+
+    /// Billing information.
+    required BillingDetails billingDetails,
```

Comment:
are you sure that billingdetails are required? I think for sepa they can be nullable.